### ISSUE TITLE
Multistore Menu is not displayed when Caching enabled

### ISSUE THREAD

**Issue by khouloudbelguith** [PrestaShop/PrestaShop#21318]
#### Describe the bug

When Caching is enabled (CacheMemcached ou CacheApc), the multistore menu is not displayed

#### Steps to Reproduce

1. Go to BO > Shop Parameters > General 
2. Enable Multistore => Multistore is added to the menu
![image](https://user-images.githubusercontent.com/16067358/95178309-92ebb180-07bf-11eb-829f-37230cf032bc.png)
3. Go to BO > Advanced Parameters > Performances 
4. Enable Caching
5. see error => multistore is not displayed
![image](https://user-images.githubusercontent.com/16067358/95178463-c0385f80-07bf-11eb-868f-f0d41ae8f226.png)

SCreenshots
1770rc1=> NOK
https://drive.google.com/file/d/1vjjUSauIbOzCQEaXn8kYL9wVG9LjDfTE/view?usp=sharing
1768 => OK
https://drive.google.com/file/d/1Nks0JRmAuFyFIo1tqDhqye413NwrMVWP/view?usp=sharing
#### Additional information

* PrestaShop version: 1.7.7.0rc1
* PHP version: 7.2

**Comment by khouloudbelguith** [comment]
Hi!
We can access to the Multistore with this URL: `admin-dev/index.php?controller=AdminShop`
But we have an alert
![image](https://user-images.githubusercontent.com/16067358/95180475-5077a400-07c2-11eb-88a3-61e4602d809a.png)

As discussed with @SimonGrn and @PrestaShop/qa-functional, it is a Major issue.

Thanks!

**Comment by atomiix** [comment]
@khouloudbelguith I've been able to reproduce it with 1.7.6.8, so I don't think this is a regression. Also, the right url to manually access the tab is `/admin-dev/index.php?controller=AdminShopGroup` and in my case, it's working.

**Comment by khouloudbelguith** [comment]
Thanks @atomiix!
I tried with PS1768 on the same server in my case => ok
https://drive.google.com/file/d/1Nks0JRmAuFyFIo1tqDhqye413NwrMVWP/view?usp=sharing
>manually access the tab is /admin-dev/index.php?controller=AdminShopGroup

You don't have any alert?

Thanks!

**Comment by atomiix** [comment]
@khouloudbelguith I think the bug is not exactly that the multistore tab is not showing when cache is enable but rather that once cache is enable, multistore tab status won't change even if you enable/disable multistore. On your 1768 with cache enable, if you try to disable multistore you should see the multistore tab still visible.

>You don't have any alert?

No alert at all!

**Comment by khouloudbelguith** [comment]
@atomiix,

>if you try to disable multistore you should see the multistore tab still visible.

Yes, the multistore is still displayed after disabling the multistore context.
![image](https://user-images.githubusercontent.com/16067358/95203052-8036a400-07e2-11eb-98c1-d67b3918f4c1.png)

Thanks!

**Comment by khouloudbelguith** [comment]
I removed the regression label so it should be removed from 177 backlog
Thanks, @atomiix!
